HackathonSampleApp/AppDelegate.swift

```diff
@@ -15,6 +15,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
```

Comment:
<!--
  0 failure: 
  1 warning:  Limit vertical wh...
  
  
  DangerID: danger-id-Danger;
  File: HackathonSampleApp/AppDelegate.swift;
  Line: 15;
-->

- :warning: Limit vertical whitespace to a single empty line. Currently 3. (`vertical_whitespace`)